lib/data/vehicle.rb

```diff
@@ -28,7 +28,8 @@ def self.add_conditions_to_join(join_query, params)
       end
 
       def self.get_random_vehicle
-        ::Vinbot::Data::Vehicle.order(Sequel.lit('RANDOM()')).limit(1).first
+        count = ::Vinbot::Data::Vehicle.count
+        ::Vinbot::Data::Vehicle.where(id: rand(count)).first
```

Comment:
I am not sure that this is guaranteed to work. If a `Vehicle` has been deleted from the database, then it will leave a gap in the list of `ids` which will cause this to return `nil` instead of a record. The new solution will run 2 queries. It looks like this could be accomplished with a single query.

http://stackoverflow.com/a/25577054
https://hashrocket.com/blog/posts/rails-quick-tips-random-records
http://thinkingeek.com/2011/07/04/easily-select-random-records-rails/

Looking around, it seems like the best approach is closer to what was originally there.
`::Vinbot::Data::Vehicle.order('RANDOM()').first` or `::Vinbot::Data::Vehicle.order('RAND()').first`, depending on the database, should accomplish the task with a single query.